backend/compact-connect/lambdas/staff-users/utils.py

```diff
@@ -0,0 +1,263 @@
+import json
```

Comment:
I feel like I've seen this code, and some of the code from other staff-user files. Is that true? Are these copies, or is the code being moved?

Reply:
Copies. A lot of the code in here is nearly or completely identical to the code in the `provider-data-v1` lambda. We can't (easily) reuse the code across those lambdas because they are separate run-times, so our options are:
1) Copy/paste some common code across the independent runtimes or
2) Add extra infrastructure and complexity specifically for the purpose of sharing common code across lambdas

Option (2) comes with a high level of additional complexity in infrastructure and especially in how we automate testing of the lambdas with multiple source bundles. It's doable but, so long as the shared/duplicated code is not huge, it probably isn't worth going down that road.

Reply:
Totally agree with your assessment. Just wanted to validate my deja vu.